<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1053</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="myhexeditor" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>530</width>
      <height>451</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>33</red>
          <green>255</green>
          <blue>6</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>33</red>
          <green>255</green>
          <blue>6</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>127</red>
          <green>127</green>
          <blue>127</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>237</red>
          <green>237</green>
          <blue>237</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>30</y>
      <width>241</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Move Cursor</string>
    </property>
    <widget class="QPushButton" name="moveBtn">
     <property name="geometry">
      <rect>
       <x>152</x>
       <y>31</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Move</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Cursor Position</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="moveCursorBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>160</y>
      <width>241</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Select Byte(s)</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Position</string>
     </property>
    </widget>
    <widget class="QPushButton" name="selectBtn">
     <property name="geometry">
      <rect>
       <x>142</x>
       <y>30</y>
       <width>91</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>Select</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>End Position</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Length</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>340</y>
      <width>241</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Insert Byte(s)</string>
    </property>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="insertBtn">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>81</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>Insert</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="clearContentBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openFileBtn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>470</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Open/Insert File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveBtn">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>470</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_2">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>90</y>
      <width>201</width>
      <height>301</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="summaryBtn">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>30</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh Buffer Info</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>400</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Output Message :</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="errorBox">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>440</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="cleanErrMsgBtn">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>400</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>clean</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>myhexeditor</class>
   <extends>QTextEdit</extends>
   <header>myhexeditor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
